codecov.yml

```diff
@@ -1,3 +1,7 @@
 coverage:
   round: down
-  precision: 0
\ No newline at end of file
+  precision: 0
```

Comment:
unrelated to this ticket, just changing to track patch codecov rather than project, as per our approach elsewhere